browser/css/cool.css

```diff
@@ -900,3 +900,54 @@ nav.spreadsheet-color-indicator ~ #sidebar-dock-wrapper {
 #about-dialog-info {
 	flex-direction: column;
 }
+
+#datepicker .ui-state-default {
+	background: #f7f7f7 ;
+	font-weight: normal;
+	color: black;
+	border: 1px solid #fff;
+}
+
+#datepicker .ui-widget-content {
+	background: #fff;
+}
```

Comment:
This is not needed we can safely change the colour directly in the jquery-ui-lightness.css`:896`, ideally to `var(--color-background-lighter)`